packages/frontend/web/src/index.tsx

```diff
@@ -74,3 +74,7 @@ try {
 } catch (err) {
   console.error('Failed to bootstrap app', err);
 }
+
+window['affine-worker'] = new Worker(new URL('./worker.ts', import.meta.url), {
```

Comment:
need a wrapper for worker thread, electron may need a different implementation. @pengx17 check it

Reply:
+1. I think we can wrap the indexer messages into RPCs and expose them into existing `apis` handlers for ease of indexing in electron later.